dags/probe_scraper.py

```diff
@@ -13,17 +15,37 @@
     'retry_delay': timedelta(minutes=30),
 }
 
-dag_daily = DAG('probe_scraper',
+with DAG('probe_scraper',
                 default_args=default_args,
-                schedule_interval='@daily')
+                schedule_interval='@daily') as dag:
+
+    probe_scraper = EMRSparkOperator(
+        task_id="probe_scraper",
+        job_name="Probe Scraper",
+        execution_timeout=timedelta(hours=4),
+        instance_count=1,
+        email=['telemetry-client-dev@mozilla.com', 'gfritzsche@mozilla.com', 'aplacitelli@mozilla.com', 'frank@mozilla.com'],
+        env={},
+        uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/probe_scraper.sh",
+        output_visibility="public",
+        dag=dag)
+
+    schema_generator = kubernetes_pod_operator.KubernetesPodOperator(
```

Comment:
consider using gkepodoperator, which is derived from k8spodoperator.  We are already using it in some dags
e.g. https://github.com/mozilla/telemetry-airflow/blob/master/dags/mango_log_processing.py#L194 and Jason has overridden some methods to fix a permissions issue
https://github.com/mozilla/telemetry-airflow/blob/master/dags/operators/gcp_container_operator.py